pkg/volume/plugins.go

```diff
@@ -743,3 +744,21 @@ type dummyPluginProber struct{}
 
 func (*dummyPluginProber) Init() error                          { return nil }
 func (*dummyPluginProber) Probe() (bool, []VolumePlugin, error) { return false, nil, nil }
+
+// getBusyboxImage returns multi-arch busybox docker image
+func getBusyBoxImage() string {
+	switch arch := runtime.GOARCH; arch {
+	case "amd64":
+		return "busybox"
```

Comment:
These should all be versioned.

Reply:
This configuration feels like something that should be externalized, rather than baked into the code...

Reply:
@bowei @tallclair For customization, use thse two flags `--pv-recycler-pod-template-filepath-nfs` and `--pv-recycler-pod-template-filepath-hostpath`.

This is just the default image name if the flags are not set.